src/wallet/db.cpp

```diff
@@ -463,15 +463,6 @@ void CDBEnv::CloseDb(const std::string& strFile)
     }
 }
 
-bool CDBEnv::RemoveDb(const std::string& strFile)
```

Comment:
Not 100% sure about this one. We don't use it, and haven't used it for a long time. From what I see it was introduced in eed1785f701be93ac2464e854c2a7de1f748ef84 (pull# unknown) and never used. Would deleting wallets be something we'd ever support? (e.g. as part of multiwallet). It is an extremely dangerous thing to do, on the other hand for watch-only wallets it'd be useful.